### PR TITLE
change(docs): Refactor the system requirements

### PR DESCRIPTION
**Pull request by upbqdn**
## Motivation

We're about to publish a blog post where we reference the system requirements. The docs for these are currently inconsistent and outdated.

## Solution

- Rename the `Zebra System Requirements` in the book to `System Requirements`.
- Refactor the `System Requirements` section in the book.
  - Remove the part about testing `lightwalletd` since that's documented elsewhere in the book.
- Simplify the main `README.md` file by moving the docs related to system requirements to the `System Requirements` section in the book.
-  Reference the `System Requirements` section from `README.md` where appropriate.

The result is that all system requirements, including some troubleshooting, are now in one section of the book, and `README.md` references this section twice:

- when describing how to build Zebra, and
- when describing the network ports that Zebra uses.

I thought the network ports merited an explicit mention in `README.md`, so I kept them there.

### Reviewer Checklist

  - [ ] Will the PR name make sense to users?
    - [ ] Does it need extra CHANGELOG info? (new features, breaking changes, large changes)
  - [ ] Are the PR labels correct?
  - [ ] Does the code do what the ticket and PR says?
    - [ ] Does it change concurrent code, unsafe code, or consensus rules?
  - [ ] How do you know it works? Does it have tests?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +68,12 @@ Zebra validates blocks and transactions, but needs extra software to generate th
 
 ## Getting Started
 
-You can run Zebra using our Docker image.
+You can run Zebra using our Docker image or you can build it manually. Please
+see [the Book](https://zebra.zfnd.org/user/requirements.html) for system
```
Comment (oxarbitrage) [inline README.md]:
```suggestion
see the [requirements section of the  Zebra Book](https://zebra.zfnd.org/user/requirements.html) for system
```

File: README.md [hunk 2]
```diff
@@ -176,103 +176,16 @@ See the [RPC config documentation](https://doc.zebra.zfnd.org/zebra_rpc/config/s
...
-
-Zebra's state commits changes using RocksDB database transactions.
-
-If you forcibly terminate Zebra, or it panics,
-any incomplete changes will be rolled back the next time it starts.
-
-So Zebra's state should always be valid, unless your OS or disk hardware is corrupting data.
+Please see the [Network
+Requirements](https://zebra.zfnd.org/user/requirements.html#network-requirements-and-ports)
+section for more details.
```
Comment (oxarbitrage) [inline README.md]:
```suggestion
section of the Zebra book for more details.
```

**Review by oxarbitrage — DISMISSED** [review]
Looks good to me, a few optional suggestions.

**Comment by upbqdn** [comment]
There was a small merge conflict, which I manually resolved.

**Review by oxarbitrage — APPROVED** [review]

**Pull request merged**